### PR TITLE
Fix ant build

### PR DESCRIPTION
**Pull request by nelsonong**

### CODE REVIEW

**Comment by ddebrunner** [comment]
Seems like the merge went wrong, the build.xml has two projects in it, ant requires a single root project.

**Comment by nelsonong** [comment]
Ah my bad, fixed the merge error.

File: com.ibm.streamsx.slack/build.xml [hunk 1]
```diff
@@ -2,6 +2,9 @@
 <!--
 # Licensed Materials - Property of IBM
 # Copyright IBM Corp. 2017, 2017
+# US Government Users Restricted Rights - Use, duplication or
```
Comment (ddebrunner) [inline com.ibm.streamsx.slack/build.xml]: Not the correct copyright for IBM code on github, lines 3,4 are sufficient.

**Comment by ddebrunner** [comment]
-1 until copyright it fixed.

**Comment by nelsonong** [comment]
Ok, fixed the copyright

**Pull request merged**